src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,102 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import mockVideos from '../../mock/youtube-videos-mock.json';
+import Switch from '../../components/Switch';
+// import { useAuth } from '../../providers/Auth';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import { NavBar, Container, VideoCard } from './Home.styled';
 
 function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+  // const history = useHistory();
+  // const { authenticated, logout } = useAuth();
+  const [active, setActive] = useState(false);
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+  // function deAuthenticate(event) {
+  //   event.preventDefault();
+  //   logout();
+  //   history.push('/');
+  // }
+
+  const lightTheme = {
+    bg: 'white',
+    fg: '#272727',
+    secondaryFg: '#3a3a3a',
+    accent: '#1b9b9b',
+    hoverAccent: '#30C8C9',
+    border: '#dddddd',
+  };
+
+  const darkTheme = {
+    bg: '#1f2127',
+    fg: 'white',
+    secondaryFg: '#b3b3b3',
+    accent: '#30C8C9',
+    hoverAccent: '#1b9b9b',
+    border: '#3a3a3a',
+  };
```

Comment:
This theme variables can be extracted into a file , e.g `theme.js` , right now they are only accessed by `Home` page but in the future, login and the video player will use them too and that will help.